BaseFilter/sections/specific.txt

```diff
@@ -280,6 +280,8 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+planetf1.com##.ps-block-a.ps-block-a-top
+planetf1.com##aside > .ps-block-a
```

Comment:
Like in prevous pull request, this will be enough. Please check this out and add the domain to the current rule if it's ok.
```adblock
planetf1.com##.ps-block-a
```